corehq/util/tests/test_quickcache.py

```diff
@@ -283,3 +283,10 @@ def by_name(name, skip_cache=False):
             @skippable_quickcache(['name'], skip_arg='missing')
             def by_name(name):
                 return 'VALUE'
+
+    def test_dict_arg(self):
```

Comment:
seems like this test could be beefed up (test a dict with multiple items in it, test a mismatch)